.github/actions/build-nextjs-website/action.yaml

```diff
@@ -62,5 +72,8 @@ runs:
         NEXT_PUBLIC_COGNITO_USER_POOL_ID: ${{ inputs.cognito_user_pool_id }}
         NEXT_PUBLIC_COGNITO_IDENTITY_POOL_ID: ${{ inputs.cognito_identity_pool_id }}
         NEXT_PUBLIC_WEBINAR_QUESTION_LIFETIME_IN_SECONDS: ${{ inputs.webinar_question_lifetime_in_seconds }}
+        STRAPI_ENDPOINT: ${{ inputs.strapi_endpoint }}
+        STRAPI_API_TOKEN: ${{ inputs.strapi_api_token }}
+        FETCH_FROM_STRAPI: ${{ inputs.fetch_from_strapi }}
```

Comment:
You should add environment's variables to `.env.default`

Reply:
Done in commit [3233097](https://github.com/pagopa/developer-portal/pull/615/commits/3233097c3ab7df9d7a2ce4f0a5047db7ed159621)